dev_guide/templates.adoc

```diff
@@ -51,67 +56,154 @@ A template describes a set of resources intended to be used together that can be
+has been uploaded to your project or global template library, using the
+Management Console.
 
-There is also the ability to add labels in the template from the command line.
+Create a template JSON file, like the above example, then upload it with the
+CLI using the following process:
+
+=== Uploading a Template
+You can upload a template to your current project's template library by passing
+a JSON file with the following command:
 
 ****
-`$ osc process -f _<filename>_ -l name=otherLabel`
+`osc create -f _<filename>_`
 ****
 
-== Parameters
-The list of parameters that you can override are listed in the `*parameters*` section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+You can upload a template to a different project using the `-n` option with the
+name of the project:
 
 ****
-`$ osc process --parameters -f _<filename>_`
+`osc create -f _<filename>_ -n _<project>_`
 ****
 
-The following shows the output when listing the parameters for one of the *_openshift/origin/examples_* templates.
-
-.Example Template Output
-====
+The template is now available to be selected for a configuration using the
+Management Console or the CLI.
 
-----
-$ osc process --parameters -f examples/sample-app/application-template-dockerbuild.json
-NAME                  DESCRIPTION              GENERATOR           VALUE
-ADMIN_USERNAME        administrator username   expression          admin[A-Z0-9]{3}
-ADMIN_PASSWORD        administrator password   expression          [a-zA-Z0-9]{8}
-MYSQL_ROOT_PASSWORD   database password        expression          [a-zA-Z0-9]{8}
-MYSQL_DATABASE        database name                                root
-----
+Alternatively, you can upload a template then create from the same template by
+piping both commands:
 
-====
-
-The output identifies several parameters that are generated with a regex expression generator when the template is processed.
-
-== Generating a Configuration
-When OpenShift processes a template, it creates a configuration. You can override the parameters with the values defined in the configuration file by adding the appropriate flags.
+****
+`osc process -f _<filename.json>_ | osc create -f -`
+****
 
+=== Generating a Configuration
 Generate a configuration with the following command:
 
 ****
 `osc process -f _<filename>_`
 ****
 
-Continuing with the previous example, you can override the *`ADMIN_USERNAME`* and *`MYSQL_DATABASE`* environment variables to create a JSON string to apply to the server. The JSON string represents a configuration with customized environment variables, as shown in the following example.
+You can override any link:../dev_guide/templates.html#parameters[parameters]
+defined in the JSON configuration file by adding the `-v` option and any desired
+parameters. For example, you can override the *`ADMIN_USERNAME`* and
+*`MYSQL_DATABASE`* parameters to create a JSON string with customized
```

Comment:
Ok. I think I know what you mean. I've taken out the few references where it says JSON string, and worded around it so it sounds ok.